spec/features/show_page_buttons_spec.rb

```diff
@@ -0,0 +1,46 @@
+# Generated via
+#  `rails generate hyrax:work Pdf`
+require 'rails_helper'
+include Warden::Test::Helpers
+
+# NOTE: If you generated more than one work, you have to set "js: true"
+RSpec.feature 'Create a PDF', :clean, js: true do
```

Comment:
Is this the correct description for the test? I'm not certain whether we are testing PDF creation, or the show page buttons.

If the latter, let's create the work with a factory, and avoid the browser interactions for work creation. If the former, we can just rewrite the feature description.